apstools/devices/positioner_soft_done.py

```diff
@@ -191,6 +191,9 @@ def _setup_move(self, position):
         if self.actuate is not None:
             self.log.debug("%s.actuate = %s", self.name, self.actuate_value)
             self.actuate.put(self.actuate_value, wait=False)
+        # This is needed because in a special case the setpoint.put does not
+        # run the "sub_value" subscriptions.
+        self.cb_setpoint()
```

Comment:
Will this callback be called twice in some cases?

Reply:
The `cb_setpoint()` method can inspect the `*args, **kwargs` to learn how it was called.

Reply:
These two lines could be run twice: https://github.com/BCDA-APS/apstools/blob/e91fa01a6797766d4a72c367917ef45a3fb3f4e3/apstools/devices/positioner_soft_done.py#L146-L147

The first looks like that would have no consequence, the second could stop a move.

Reply:
I think it will not stop the motion. It could false-flag the motion as not done, but the `self.cb_readback()` in the following line forces a position check.